packages/eslint-plugin/tests/rules/no-identifier-enum-member.test.ts

```diff
@@ -0,0 +1,44 @@
+import rule, { MessageId } from '../../src/rules/no-identifier-enum-member';
+import { RuleTester } from '../RuleTester';
+
+const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
+});
+
+ruleTester.run('prefer-literal-enum-member', rule, {
+  valid: [
+    `
+enum Valid {
+  A = /test/,
+  B = 'test',
+  C = 42,
+  D = null,
+  E,
+  F = {},
+  G = [],
+  H = new Set(),
+  I = 2 + 2,
+}
+    `,
+  ],
```

Comment:
please add sanity checks for the following cases:
```ts
enum Foo {
  'a' = 1,
  'a',
  ['a'],
}
```

Reply:
will do.